cmd/resource-topology-exporter/main.go

```diff
@@ -108,14 +152,15 @@ func getUsage() (string, error) {
 
 // nrtupdaterArgsParse parses the command line arguments passed to the program.
 // The argument argv is passed only for testing purposes.
-func argsParse(argv []string) (nrtupdater.Args, resourcemonitor.Args, resourcetopologyexporter.Args, error) {
+func argsParse(argv []string) (nrtupdater.Args, resourcemonitor.Args, resourcetopologyexporter.Args, localArgs, error) {
 	var nrtupdaterArgs nrtupdater.Args
 	var resourcemonitorArgs resourcemonitor.Args
 	var rteArgs resourcetopologyexporter.Args
+	var args localArgs
```

Comment:
We have a bunch of different types of args here. I would prefer it to have a more explicit name like the others.
```var localArgs localArgs``` is ok I guess.

Reply:
sure, done.